editor/test/integration/gui_test.clj

```diff
@@ -365,9 +385,9 @@
                   (gui-node gui-node-name)
                   (select-labels labels)))]
 
-      (let [button-gui-resource (workspace/find-resource workspace "/main/button.gui")
-            panel-gui-resource (workspace/find-resource workspace "/main/panel.gui")
-            window-gui-resource (workspace/find-resource workspace "/main/window.gui")
+      (let [button-gui-resource (workspace/find-resource workspace "/gui/template/button.gui")
+            panel-gui-resource (workspace/find-resource workspace "/gui/template/panel.gui")
+            window-gui-resource (workspace/find-resource workspace "/gui/template/window.gui")
```

Comment:
I moved these files in the gui test project because I added a bunch of new scenes to it and felt it needed reorganising a bit.